src/main/java/org/mastodon/views/trackscheme/display/PaintBranchGraph.java

```diff
@@ -43,7 +43,7 @@
 public class PaintBranchGraph extends PaintGraph
 {
 
-	private final Stroke edgeStroke = new BasicStroke( 1.5f );
+	protected Stroke edgeStroke = new BasicStroke( 1.5f );
```

Comment:
What about we define a stroke for 
- normal TrackScheme
- branch TrackScheme
- hierarchy TrackScheme
in the `TrackSchemeStyle`? 
Possibly we could later let the user edit them.

Reply:
Yes, it should be done as you suggest.